### ISSUE TITLE
DXUP does not compile on Linux

### ISSUE THREAD

**Issue by ronoverdrive** [Tk-Glitch/PKGBUILDS#93]
In response to commit 0706dad, I have to point out that DXUP at this time does not compile on Linux. Just in case you came up with some magic patches for it I tried the option and confirmed that it doesn't compile with your PKGBUILD. Josh does offer pre-built binaries via [appveyor](https://ci.appveyor.com/project/Joshua-Ashton/dxup) that should work if you want to keep this option available.

**Comment by Tk-Glitch** [comment]
It does work if you have followed the required steps described in the commit adding support for DXUP.
```
you'll have to build WITHOUT dxup first
so your wine can THEN build dxup successfully
```

**Comment by Tk-Glitch** [comment]
The required patch for wine's d3d9 headers will be applied on any build made after the DXUP support commit, but since you need your currently installed build to have it to be able to build DXUP.. Well you got it.

**Comment by clapbr** [comment]
Whats the current state of dxup? any playable games?

**Comment by Tk-Glitch** [comment]
@clapbr Very experimental. Probably no playable game at this point (or at least not without major issues). But it's moving quickly.

**Comment by Joshua-Ashton** [comment]
It can play HL2 if you're OK with frogged up bones on characters.

**Comment by clapbr** [comment]
we all like any frog things here

**Comment by Tk-Glitch** [comment]
![screenshot_20190121_131659](https://user-images.githubusercontent.com/741977/51474225-ee250880-1d7e-11e9-9d3d-c981a78eee44.png)
Don't mind the hud, it was actually running at 300 fps

**Comment by Joshua-Ashton** [comment]
It's me Gordon! Barney from Black Mesa. Sorry for the scare, I had to put on a show for the cameras. I've been working undercover with Civil Protection -- I can't take to long or they'll get suspicious, I'm waaay behind on my beating quota.

**Comment by Joshua-Ashton** [comment]
Yes Barney what is it? I'm in the middle of a critical test...

**Comment by Joshua-Ashton** [comment]
Sorry Doc but... look who's here!

**Comment by Joshua-Ashton** [comment]
GREAT SCOTT! GORDON FREEMAN! ID HAVE EXPECTED MORE WARNING

**Comment by Joshua-Ashton** [comment]
Yeah you and me both Doc, he was about to board the express to Nova Prospekt...

**Comment by ryanmusante** [comment]
Black Mesa through wine?
